ios/Podfile.lock

```diff
@@ -1080,7 +1080,7 @@ SPEC CHECKSUMS:
   Kronos: 0ca73a1c00dc9a49a0bab4a6ed42f282562a7058
   leveldb-library: 50c7b45cbd7bf543c81a468fe557a16ae3db8729
   libevent: 4049cae6c81cdb3654a443be001fb9bdceff7913
-  Mapbox-iOS-SDK: 6a67397df2bcafe5d2851604192aa8de6f8d6c89
+  Mapbox-iOS-SDK: 2563ed87ead6ec08f1c2090873977b8a7be335a8
```

Comment:
Jeg tror dette blir feil forøvrig. Det er en issue hvor det er ulike checksums som brukes for Mapbox.. 🤔  Ikke tatt tid til å finne ut hvorfor (les: greide ikke finne ut av det)

Reply:
Får tilsvarende checksum på min egen branch, så er noe muffens et sted

Reply:
Er det et problem? 
Er mest irritasjon hvis den endrer seg for hver enkelt.